### PR TITLE
Issue #74: Fix filter default and use correct createTask

### PR DESCRIPTION
**Pull request by ivanrobledofutit**
ETP-2900-Y24

### CODE REVIEW

File: src-test/src/com/etendoerp/task/utils/TaskUtilTest.java [hunk 1]
```diff
@@ -613,8 +600,8 @@ public void testCreateTaskMissingClient() throws Exception {
    */
   @Test
   public void testGetInitialStateFound() {
```
Comment (etendobot) [inline src-test/src/com/etendoerp/task/utils/TaskUtilTest.java:602]: **Suggestion**

Consider renaming the test method to explicitly mention `null` since the current scenario only exercises a `null` filter value. If `""` (empty) is also expected to behave as true, you might add a second test covering that case explicitly.

```java
  @Test
  void testValidateFilterWithNullFilterReturnsTrue() throws Exception {
    String filter = null;
    JSONObject data = new JSONObject().put("anyField", "anyValue");

    boolean result = TaskUtil.validateFilter(filter, data);

    assertTrue(result);
  }
```

**Rationale**: Etendo test naming patterns usually try to reflect the exact scenario being validated. Making the name explicit (or adding a complementary test for the empty-string case) improves test readability and ensures both expected behaviors (null and empty) are covered if required by the business rule.

**Review by etendobot — APPROVED** [review]

**Review by etendobot — CHANGES_REQUESTED** [review]
Changes requested by agent. Please resolve blocking issues.

**Review by valeg-etendo — APPROVED** [review]

**Review by jortolanosmf — APPROVED** [review]
